src/test/java/org/mule/service/soap/TestSoapClient.java

```diff
@@ -101,6 +113,7 @@ public void start() {
 
   @Override
   public void stop() {
+    httpClient.stop();
```

Comment:
Use FunctionalUtils.safely() so that if this invokation fails, the dispatcher is also disposed